.eslintrc.json

```diff
@@ -23,7 +23,8 @@
     "comma-dangle": "off",
     "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
     "react/prop-types": "off",
-    "semi": "off"
+    "semi": "off",
+    "linebreak-style": [0 ,"error", "windows"]
```

Comment:
I haven't really worked with the `linebreak-style` rule before, but from what I gathered from the [ESLint documentation](https://eslint.org/docs/latest/rules/linebreak-style), this rule will enforce the use of CRLF and I don't think we want that. The documentation says that the default behavior for git on Windows is to convert LF to CRLF when checking out files, but to store them as LF when actually committing the change (which is why you don't see any of the line break warnings on Github when you push anything). The documentation mentions adding a `.gitattributes` file when using version control, so maybe we can look into using that (if it's applicable here) instead of adding this rule, or see if we can disable the CRLF conversion locally.

Reply:
I found a command to run in gitbash `npm run lint -- --fix` that seems to fix the eol issues on an individual machine without interfering with other people's environments. I did run into a warning message: **"warning: LF will be replaced by CRLF. The file will have its original line endings in your working directory"** but after looking into it, a bunch of people have said that not only is the warning backwards in it's wording (which was ever so confusing) but it is also irrelevant because it is just telling you that what you want to happen is happening. I looked into a few different ways to fix it instead of running this command but none of the others worked, so if this doesn't look like a good solution either then I think that a `.gitattributes` file would be the next best bet.